js/store.js

```diff
@@ -320,6 +321,10 @@ const store = {
     // merge data with thirdParty
     // @todo restructure storage to use data from capture for graph filtering
     for (const key in data) {
+      if (key ==='firstParty' && this.isAlsoAFirstParty(thirdParty)) {
```

Comment:
Nit space after `===`

Reply:
Fixed, thank you... is there an eslint rule for that?

Reply:
Probs consistent whitespace one. Feel free to file a bug, right now don't have time to check this.

Reply:
Okay I found it; it's `space-infix-ops`. I added it to `eslintrc.json`.